docs/references/customization/staging-scripts.md

```diff
@@ -0,0 +1,120 @@
+---
+title: ""
+sidebar_label: "Staging Scripts"
+---
+
+# Customization point: Staging Scripts
+
+## Overview
+
+:::note
+This customization interacts with the
+[customization of buildpacks](staging.md).
+:::
+
+Staging scripts are what Epinio uses to interface with the
+Paketo [Cloud Native Buildpacks](https://buildpacks.io/).
+
+Epinio automatically selects the set of staging scripts based on the name of the chosen builder
+image, and the images supported by a specific definition.
+
+By default Epinio installs three definitions.
+
+  1. One to support Bionic-based builder images (`paketo-buildpacks/builder:*`).
+  2. One to support Jammy-based  builder images (`paketo-buildpacks/builder-jammy-*:*`).
+  3. And one to serve as fallback for any images not captured by the other two.
```

Comment:
```suggestion
  1. One to support Bionic-based builder images (`paketo-buildpacks/builder:*`).
  1. One to support Jammy-based  builder images (`paketo-buildpacks/builder-jammy-*:*`).
  1. One to serve as fallback for any images not captured by the other two.
```

Reply:
Is it not better to use bullet points instead of a numeric list in this case?

Reply:
It might be a style-preference. I liked numeric list better for this.